mesonbuild/minstall.py

```diff
@@ -98,7 +136,7 @@ def set_mode(path, mode, default_umask):
     # No chown() on Windows, and must set one of owner/group
     if not is_windows() and (mode.owner or mode.group) is not None:
         try:
-            shutil.chown(path, mode.owner, mode.group)
+            set_chown(path, mode.owner, mode.group, follow_symlinks=False)
```

Comment:
Probaby here we could have `follow_symlinks=os.file.isfile(path)`, however I think it doesn't really make much sense (not to say is wrong) to apply on the linked file.